pkg/swarm/swarm.go

```diff
@@ -23,7 +23,7 @@ const (
 	BmtBranches             = 128
 	ChunkSize               = SectionSize * Branches
 	HashSize                = 32
-	MaxPO             uint8 = 15
+	MaxPO             uint8 = 31
```

Comment:
Please read issue #1577 as this change will break cursor retrieval for pullsync.